### PR TITLE
feat(jandan): add new route for girl gallery

### PR DESCRIPTION
**Pull request by bxb100**
## Example for the Proposed Route(s) / 路由地址示例

```routes
/jandan/girl
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

File: lib/routes/jandan/girl.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    // set debug info
+    ctx.set('json', data);
+    return data;
+}
+
+const __dirname = getCurrentPath(import.meta.url);
+const girlDescTemplate = path.join(__dirname, 'templates/girl.art');
+
+async function _crawl(startId?: string, limit: number = 30, items: DataItem[] = []): Promise<DataItem[]> {
+    if (items.length >= limit) {
```
Comment (TonyRL) [inline lib/routes/jandan/girl.ts:40]: No page turn.

Comment (bxb100) [reply lib/routes/jandan/girl.ts:40]: what did you mean?

Comment (TonyRL) [reply lib/routes/jandan/girl.ts:40]: https://github.com/DIYgod/RSSHub/blob/60dc249fef0093ed8d20c346d6b9f99436bdfaa6/lib/routes/jandan/girl.ts#L63-L65

Comment (bxb100) [reply lib/routes/jandan/girl.ts:40]: there are no extra parameters in Jandan's API

**Pull request closed without merging**